src/crate/theme/rtd/conf/__init__.py

```diff
@@ -113,7 +113,7 @@
 
 # Configure OpenGraph extension
 ogp_site_url = "https://crate.io/"
-ogp_description_length = 300
+ogp_description_length = 150
```

Comment:
Just for the records. Can you elaborate again why this change was made?

Reply:
@amotl Even though there's no real length limit (there might be one at 10k chars), 150 - 160 characters is the most popular recommendation for the og description (which is also the recommended length for the regular meta description).

Like https://www.opengraph.xyz/ shows popular services and the corresponding description